src/components/Header/index.tsx

```diff
@@ -42,6 +42,9 @@ const Header = ({ t }: { t: TFunction }) => {
         <CustomNavLinkSmall onClick={() => scrollTo("product")}>

```

Comment:
- **Runtime error check**  
The `scrollTo` function is not checked for null or undefined before being called. This could lead to a runtime error if `scrollTo` is null or undefined.  
- **Optimization**  
  No optimization points found.  
- **Security issue**  
  No security issues found.